jenkins_jobs/trigger_weekly_host_os_build/script.sh

```diff
@@ -53,19 +60,19 @@ upgrade_versions() {
            upgrade-versions \
                --updater-name "$GITHUB_BOT_NAME" \
                --updater-email "$GITHUB_BOT_EMAIL" \
-               --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
+               --push-repo-url "$VERSIONS_PUSH_URL" \
                --push-repo-branch "$COMMIT_BRANCH"
 }
 
 create_release_notes() {
     python host_os.py \
            --verbose \
            release-notes \
-               --build-versions-repository-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
+               --build-versions-repository-url "$VERSIONS_PUSH_URL" \
```

Comment:
This could be the HTTPS URL, but ok, there's no need to have two URLs for the versions repo.